<ui version="4.0" >
 <class>MainWindow1</class>
 <widget class="QWidget" name="MainWindow1" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MyCash[*]</string>
  </property>
  <widget class="QTabWidget" name="tabWidgetMain" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>40</y>
     <width>1211</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="Seite" >
    <attribute name="title" >
     <string>Seite</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>353</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgramm" >
    <property name="title" >
     <string>program</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_ber" >
    <property name="title" >
     <string>about</string>
    </property>
    <addaction name="actionMyCash" />
    <addaction name="actionQt" />
   </widget>
   <widget class="QMenu" name="menuProject" >
    <property name="title" >
     <string>project</string>
    </property>
    <addaction name="actionNewAccount" />
    <addaction name="actionAddAccount" />
   </widget>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>settings</string>
    </property>
   </widget>
   <addaction name="menuProgramm" />
   <addaction name="menuProject" />
   <addaction name="menuSettings" />
   <addaction name="menuHelp" />
   <addaction name="menu_ber" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionNew" >
   <property name="text" >
    <string>new</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>exit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionMyCash" >
   <property name="text" >
    <string>MyCash</string>
   </property>
  </action>
  <action name="actionQt" >
   <property name="text" >
    <string>Qt</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>save as</string>
   </property>
  </action>
  <action name="actionNewAccount" >
   <property name="text" >
    <string>create new account</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAddAccount" >
   <property name="text" >
    <string>add exist account</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow1</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>639</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
